.ci/e2eKibana.groovy

```diff
@@ -7,6 +7,7 @@ pipeline {
   environment {
     REPO = 'kibana'
     BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    ELASTIC_REPO = "elastic/${env.REPO}"
```

Comment:
If you use the the gitCheckout then ORG_NAME should be there and point to `elastic`:
- https://github.com/elastic/apm-pipeline-library/blob/8f48394fe31ae516841e25ea45b3faa94d147e96/vars/githubEnv.groovy#L79-L80
In case it helps you to move away from the hardcoded elastic

Reply:
We are internally using `gitCheckout` in the buildKibana step, although it could be the case the image is already built and we skip that part